packages/next/build/entries.ts

```diff
@@ -166,6 +166,8 @@ export function getEdgeServerEntry(opts: {
   middleware?: Partial<MiddlewareConfig>
   pagesType?: 'app' | 'pages' | 'root'
   appDirLoader?: string
+  // TODO(schniz): mark as optional
```

Comment:
Is this TODO still needed? Seems the `undefined` type below should be sufficient.